src/transformStateWithClones.js

```diff
@@ -8,6 +8,39 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+
+  const arrResolt = [];
+
+  let counter = 0;
```

Comment:
this counter is redundant. create a copy of state object
```
const stateCopy = { ...state };
```

Reply:
we need the last modified object in arrResolt. not a copy of 'state'.